src/services/fetch/web_sys.rs

```diff
@@ -3,37 +3,51 @@
 use crate::callback::Callback;
 use crate::format::{Binary, Format, Text};
 use crate::services::Task;
-use thiserror::Error;
-use futures::future::{FutureExt, TryFutureExt};
-use js_sys::Reflect;
-use js_sys::{Array, Uint8Array};
+use anyhow::anyhow;
+use http::request::Parts;
+use js_sys::{Array, Promise, Reflect, Uint8Array};
+use std::cell::RefCell;
 use std::fmt;
 use std::iter::FromIterator;
+use std::marker::PhantomData;
 use std::rc::Rc;
-use std::sync::atomic::{AtomicBool, Ordering};
+use thiserror::Error;
```

Comment:
I usually do this to be able to use both without scoping all the time:
```suggestion
use thiserror::Error as ThisError;
use anyhow::Error;
```

Reply:
FYI you can just import both.

```rust
use anyhow::Error;
use thiserror::Error;
```

One is a macro and the other is a type, so those don't conflict.

Reply:
Wow, I didn't know that, thanks!